mage_ai/streaming/sources/activemq.py

```diff
@@ -0,0 +1,87 @@
+import time
+from dataclasses import dataclass
+from typing import Callable
+
+import stomp
+from stomp.exception import ConnectFailedException
+
+from mage_ai.shared.config import BaseConfig
+from mage_ai.streaming.sources.base import BaseSource
+
+
+@dataclass
+class ActiveMQConfig(BaseConfig):
+    connection_host: str
+    connection_port: int
+    queue_name: str
+    configure_consume: bool = False
+    username: str = 'admin'
+    password: str = 'admin'
+
+
+def messageProcessingFunction(message, handler):
+    print('Recieved message: "%s"' % message)
```

Comment:
what are the possible types of the message? can it be bytes, str, dict? or is it just a string?